catroid/src/main/java/org/catrobat/catroid/content/bricks/LegoEv3MotorMoveBrick.java

```diff
@@ -81,26 +77,21 @@ protected Object readResolve() {
 		return this;
 	}
 
-	private void initializeBrickFields(Formula powerFormula, Formula durationFormula) {
-		addAllowedBrickField(BrickField.LEGO_EV3_POWER);
-		addAllowedBrickField(BrickField.LEGO_EV3_PERIOD_IN_SECONDS);
-		setFormulaWithBrickField(BrickField.LEGO_EV3_POWER, powerFormula);
-		setFormulaWithBrickField(BrickField.LEGO_EV3_PERIOD_IN_SECONDS, durationFormula);
+	private void initializeBrickFields(Formula speed) {
+		addAllowedBrickField(BrickField.LEGO_EV3_SPEED);
+		setFormulaWithBrickField(BrickField.LEGO_EV3_SPEED, speed);
 	}
 
 	@Override
 	public int getRequiredResources() {
-		return BLUETOOTH_LEGO_EV3 | getFormulaWithBrickField(BrickField.LEGO_EV3_POWER).getRequiredResources()
-				| getFormulaWithBrickField(BrickField.LEGO_EV3_PERIOD_IN_SECONDS).getRequiredResources();
+		return BLUETOOTH_LEGO_EV3 | getFormulaWithBrickField(BrickField.LEGO_EV3_SPEED).getRequiredResources();
 	}
 
 	@Override
 	public View getPrototypeView(Context context) {
 		prototypeView = View.inflate(context, R.layout.brick_ev3_motor_move, null);
-
-		TextView textPower = (TextView) prototypeView.findViewById(R.id.ev3_motor_move_power_edit_text);
-
-		textPower.setText(String.valueOf(BrickValues.LEGO_POWER));
+		TextView textSpeed = (TextView) prototypeView.findViewById(R.id.ev3_motor_move_speed_edit_text);
+		textSpeed.setText(String.valueOf(BrickValues.LEGO_SPEED));
```

Comment:
why is it, that sometimes constants related to this brick are called "LEGO_EV3_SPEED" and sometimes "LEGO_SPEED"?